Microsoft.HealthVault.Fhir/Codes/HealthVault/HealthVaultVitalStatisticsCodes.cs

```diff
@@ -15,14 +16,14 @@ namespace Microsoft.HealthVault.Fhir.Codes.HealthVault
     /// </summary>
     public static class HealthVaultVitalStatisticsCodes
     {
-        public const string System = "http://healthvault.com/vocabularies/vital-statistics";
+        public static readonly string System = VocabularyUris.HealthVaultVocabulariesUri;
 
-        internal static Coding BodyWeight = new Coding()
+        internal static readonly Coding s_bodyWeight = new Coding()
```

Comment:
>s_b [](start = 40, length = 3)

Aren't we using PascalCase (without the s_ prefix) for all internal and public properties?